### ISSUE TITLE
0.24.0-alpha-2, problems during configuration of generateDebugLintReportModel

### ISSUE THREAD

**Issue by Burtan** [icerockdev/moko-resources#642]
Hi,
I'm getting the following error when building my shared module with moko-resources 0.24.0-alpha-2.

```
Some problems were found with the configuration of task ':shared:generateDebugLintReportModel' (type 'LintModelWriterTask').
  - Gradle detected a problem with the following location: '/.../template-compose-multiplatform/shared/build/generated/moko-resources/androidUnitTest/assets'.
```

Here is a [corresponding project](https://github.com/Burtan/template-compose-multiplatform) for reproduction.

**Comment by Alex009** [comment]
@ExNDY need to check reproducer project with alpha-3. i think bug should be fixed already in #575

**Comment by Burtan** [comment]
I can confirm this is fixed with 0.24.0-alpha-3!
